CMakeLists.txt

```diff
@@ -54,6 +63,20 @@ set(LIEF_ART OFF CACHE BOOL "" FORCE)
 set(LIEF_DEX OFF CACHE BOOL "" FORCE)
 set(LIEF_VDEX OFF CACHE BOOL "" FORCE)
 
+# Fetch GLFW 3.3
+FetchContent_Declare(glfw
+    GIT_REPOSITORY https://github.com/glfw/glfw.git
+    GIT_TAG 3.3.9
+)
+
+# Configure GLFW options
+set(GLFW_BUILD_DOCS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_TESTS OFF CACHE BOOL "" FORCE)
+set(GLFW_BUILD_EXAMPLES OFF CACHE BOOL "" FORCE)
+set(GLFW_INSTALL OFF CACHE BOOL "" FORCE)
+
+FetchContent_MakeAvailable(glfw)
```

Comment:
Is this fetched on Windows? If yes, is that necessary?

Reply:
No need, I've updated this so it doesn't happen on windows